packages/manager/apps/web/client/app/dns-zone/new/dns-zone-new.html

```diff
@@ -10,7 +10,10 @@
     </oui-guide-menu>
 </oui-header>
 
-<div data-ovh-alert="{{$ctrl.alerts.main}}"></div>
+<div
+    data-ng-attr-id="{{:: $ctrl.getAlerterId($ctrl.alerts.main)}}"
```

Comment:
You could use the `constructor` property to access statics members
```suggestion
    data-ng-attr-id="{{:: $ctrl.constructor.getAlerterId($ctrl.alerts.main)}}"
```